patches/net/minecraft/world/item/ProjectileWeaponItem.java.patch

```diff
@@ -1,5 +1,34 @@
 --- a/net/minecraft/world/item/ProjectileWeaponItem.java
 +++ b/net/minecraft/world/item/ProjectileWeaponItem.java
+@@ -24,12 +_,28 @@
+         super(p_43009_);
+     }
+ 
++    /**
++     * Use ItemStack sensitive version instead
++     */
++    @Deprecated
```

Comment:
Please also use the javadoc deprecated marker; it shows up as part of the mouse-over warning, while the javadoc itself may be hidden by the warning.